src/relay/ir/doc.cc

```diff
@@ -23,6 +23,7 @@
  * \brief Doc ADT used for pretty printing.
  * Based on Section 1 of https://homepages.inf.ed.ac.uk/wadler/papers/prettier/prettier.pdf.
  */
+#include <cassert>
 #include <memory>
```

Comment:
Likely we don’t need this, but need to change assert to CHECK

Reply:
Got it.